### PR TITLE
[PM-8805] migrate cli to generator libraries

### PR DESCRIPTION
**Pull request by audrey-jensen**
## 🎟️ Tracking

https://bitwarden.atlassian.net/browse/PM-8805

## 📔 Objective

Replace dependencies on `@bitwarden/common/tools/generator` with `@bitwarden/generator-core` and `@bitwarden/generator-extensions` so that the common code can be deleted.

## 🦮 Reviewer guidelines

- 👍 (`:+1:`) or similar for great changes
- 📝 (`:memo:`) or ℹ️ (`:information_source:`) for notes or general info
- ❓ (`:question:`) for questions
- 🤔 (`:thinking:`) or 💭 (`:thought_balloon:`) for more open inquiry that's not quite a confirmed issue and could potentially benefit from discussion
- 🎨 (`:art:`) for suggestions / improvements
- ❌ (`:x:`) or ⚠️ (`:warning:`) for more significant problems or concerns needing attention
- 🌱 (`:seedling:`) or ♻️ (`:recycle:`) for future improvements or indications of technical debt
- ⛏ (`:pick:`) for minor or nitpick changes

### CODE REVIEW

**Review by JaredSnider-Bitwarden — DISMISSED** [review]
Auth file change looks good!

File: apps/cli/src/tools/generate.command.ts [hunk 1]
```diff
@@ -1,15 +1,17 @@
-import { StateService } from "@bitwarden/common/platform/abstractions/state.service";
-import { DefaultPassphraseGenerationOptions } from "@bitwarden/common/tools/generator/passphrase";
+import { StateService } from "@bitwarden/common/src/platform/abstractions/state.service";
```
Comment (djsmith85) [inline apps/cli/src/tools/generate.command.ts:1]: :x: This import shouldn't change

**Review by djsmith85 — CHANGES_REQUESTED** [review]
Nice work! Only one minor thing and this is good to go

File: apps/cli/src/tools/generate.command.ts (hunk 1 above)
Comment (audrey-jensen) [reply apps/cli/src/tools/generate.command.ts:1]: The `StateService` import?

Comment (audrey-jensen) [reply apps/cli/src/tools/generate.command.ts]: Resolved in 04dafeccc.

**Review by djsmith85 — APPROVED** [review]

**Pull request closed without merging**